docs/schema.json

```diff
@@ -4170,7 +4202,8 @@
         },
         "selectedRooms": {
```

Comment:
another case: input should be an array. 
probably one addition field doesn't solve all problems